0010-endpoint-metadata-extension/README.md

```diff
@@ -0,0 +1,658 @@
+---
+name: Endpoint Metadata Extension
+shortname: 10/EPMDX
+status: draft
+editor: Volodymyr Tkhir <vtkhir@cybervisiontech.com>
+---
+
+- [Introduction](#introduction)
+- [Requirements and constraints](#requirements-and-constraints)
+  - [Metadata keys](#metadata-keys)
+  - [Metadata values](#metadata-values)
+  - [Supported operations](#supported-operations)
+- [Design](#design)
+  - [Full metadata update](#full-metadata-update)
+    - [Full metadata update request](#full-metadata-update-request)
+    - [Full metadata update response](#full-metadata-update-response)
+  - [Partial metadata update](#partial-metadata-update)
+    - [Partial metadata update request](#partial-metadata-update-request)
+    - [Partial metadata update response](#partial-metadata-update-response)
+  - [Get metadata](#get-metadata)
+    - [Get metadata request](#get-metadata-request)
+    - [Get metadata response](#get-metadata-response)
+  - [Get metadata keys](#get-metadata-keys)
+    - [Get metadata keys request](#get-metadata-keys-request)
+    - [Get metadata keys response](#get-metadata-keys-response)
+  - [Delete metadata key](#delete-metadata-key)
+    - [Delete metadata key request](#delete-metadata-key-request)
+    - [Delete metadata key response](#delete-metadata-key-response)
+
+
+## Introduction
+
+The Endpoint Metadata Extension (EPMDX) protocol is an endpoint-aware [Kaa Protocol](/0001-kaa-protocol/README.md) extension.
+
+The Endpoint Metadata Extension is intended to manage endpoint's metadata.
+The metadata provides information about the endpoint and is a collection of key-value pairs.
+Example endpoint's metadata keys are `name`, `description`, `location`, `vendor`, `device model`, `firmware version` etc.
+
+## Requirements and constraints
```

Comment:
OK, so should I remove the list of supported operations from this section and change this title to `Constraints`?